staging/src/k8s.io/api/admissionregistration/v1beta1/generated.proto

```diff
@@ -81,6 +81,17 @@ message Rule {
   // Depending on the enclosing object, subresources might not be allowed.
   // Required.
   repeated string resources = 3;
+
+  // Scope specifies the scope of this rule.
+  // Valid values are "Cluster", "Namespaced", and "*"
+  // "Cluster" means that only cluster-scoped resources will match this rule.
+  // Namespace API objects are cluster-scoped.
+  // "Namespaced" means that only namespaced resources will match this rule.
+  // "*" means that there are no scope restrictions.
```

Comment:
How about explicitly calling out subresources, "subresources inherits the scope of the main resource"?

Reply:
updated